manifest

```diff
@@ -11,13 +11,22 @@ export DOCUMENTATION_URL="https://chimeraos.org/about"
 export BUG_REPORT_URL="https://github.com/ChimeraOS/chimeraos/issues"
 
 export PACKAGES="\
-	gnome-desktop \
 	gnome-control-center \
 	gnome-session \
 	gnome-shell \
 	gnome-software \
 	gnome-terminal \
+	gnome-keyring \
+	gnome-menus \
+	gnome-system-monitor \
 	nautilus \
+	sushi \
+	eog \
+	evince \
+	file-roller \
+	gedit \
```

Comment:
Not sure about this one... you can grab any text editor from flathub.

Reply:
True. And gedit is there in flathub. But having it native doesn't add too much and is better to have it readily available than the user needing to surf the web or the software center for it. I can foresee some users trying to install software downloading from Softnic or UptoDown because of this.
The goal is to provide a minimal functional desktop with the tools to follow a tutorial on protondb or the internet devised for the Deck (mostly).

Reply:
Well, I guess we can keep it.